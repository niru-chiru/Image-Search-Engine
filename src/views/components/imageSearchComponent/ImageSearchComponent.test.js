import { render, screen } from "@testing-library/react";
import ImageSearchComponent from "./imageSearchComponent";

const data = [];

it("matches snapshot", () => {
  const { asFragment } = render(<ImageSearchComponent ImageList={data} />);
  expect(asFragment()).toMatchSnapshot();
});
