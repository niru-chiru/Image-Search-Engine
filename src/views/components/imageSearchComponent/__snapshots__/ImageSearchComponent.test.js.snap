// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="ImageSearchComponent-root-1"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    />
  </div>
</DocumentFragment>
`;
